GNUmakefile

```
@@ -19,4 +19,4 @@ test: $(TESTS)
 	lake env lean $(@:.run=.lean)
 
 lint: build
```

```suggestion
lint:
```

Lake will do the build. ;)